app/models/job_offer_actor.rb

```diff
@@ -8,22 +8,16 @@ class JobOfferActor < ApplicationRecord
   accepts_nested_attributes_for :administrator
 
   def administrator_attributes=(attributes)
-    existing = nil
-    existing = Administrator.find(attributes[:id]) if attributes[:id].present?
-    existing ||= Administrator.where(email: attributes[:email]).first
-    if existing
-      if has_destroy_flag?(attributes)
-        mark_for_destruction
-      else
-        self.administrator = existing
-      end
-    elsif has_destroy_flag?(attributes)
-      mark_for_destruction
+    mark_for_destruction if has_destroy_flag?(attributes)
```

Comment:
did this method had a test suite? if not, can you add one?

Reply:
https://github.com/betagouv/civilsdeladefense/pull/771/commits/a9be8c391f880bc86f89f8c480ab1833cd877688#diff-300e259f4866f0ba878a0ab70211af9453df4a87d60eba34ce5eaabf4774041cL155-L158

Done